lib/team_builder.rb

```diff
@@ -96,7 +96,7 @@ def govuk_json
   end
 
   def govuk_team_repos(team_channel)
-    @govuk_data.select { |repos| repos["team"] == team_channel }.map { |repo| repo["app_name"] }
+    @govuk_data.select { |repos| repos["alerts_team"] == team_channel }.map { |repo| repo["app_name"] }
   rescue StandardError => e
     puts "Error fetching govuk team repos (#{team_channel}): #{e.message}"
     []
```

Comment:
Do we need to update the `is_govuk_team?` for consistency? 🤔

Reply:
Yes, I think we do.  I've updated that in https://github.com/alphagov/seal/pull/571.